Source/LinqToDB/Data/DataConnection.Async.cs

```diff
@@ -299,7 +304,7 @@ public async Task DisposeAsync()
 					dataConnection.OnTraceConnection(new TraceInfo(dataConnection, TraceInfoStep.AfterExecute, traceOperation, true)
 					{
 						TraceLevel    = TraceLevel.Info,
-						CommandText   = commandText,
+						CommandText   = dataConnection.TraceSwitchConnection.TraceInfo ? sql : commandText?.Invoke(context),
```

Comment:
* We already know `dataConnection.TraceSwitchConnection.TraceInfo == true` due to the `if()`
* We know that when it is `true`, the value is already set in `sql`.

```suggestion
						CommandText   = sql,
```

Reply:
whoops, it should be in error report... will fix